Flow.Launcher/ViewModel/MainViewModel.cs

```diff
@@ -746,7 +746,7 @@ public void Show()
             MainWindowVisibilityStatus = true;
             
             if(_settings.UseAnimation)
-                ((MainWindow)Application.Current.MainWindow).WindowAnimator();
+                ((MainWindow)Application.Current.MainWindow)?.WindowAnimator();
```

Comment:
so if the mainwindow is null, there wont be animation but will still make the window visible or will nothing show at all?

Reply:
If it is null, something unwanted happens 🤣

Reply:
i dont think this is a necessary, when null it will not animate and just do the original show mainwindow, and MainWindow should never be null.
 ```suggestion
                ((MainWindow)Application.Current.MainWindow).WindowAnimator();
```

Reply:
ok xd. Just let it throw so we know something weird happen.